docs/stylesheets/star-rating.css

```diff
@@ -0,0 +1,143 @@
+@import url(//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css);
+
+
+#leave-feedback {
+  margin-top: 8px;
+}
+
+#starbox {
+  margin-top: -3px;
+}
+
+#stars {
+  display: block;
+  width: 125px;
+}
+
+#submitform {
+  border-radius: 4px;
+  display: none;
+  -webkit-animation: fadeIn 1 s;
+  animation: fadeIn 1 s;
```

Comment:
should be `animation: fadeIn 1s;`, without the space in `1 s`. This way the popup will fade in nicely.